### DISCUSSION TITLE
npm-merge-driver guidance

### DISCUSSION THREAD

**Discussion by mbrevda** [npm/feedback#538] [CLI]
Hi -

The [npm-merge-driver](https://github.com/npm/npm-merge-driver) has been marked as archived, but no guidance is offered as to why, nor is an alternative suggested. Is the driver "mature"? Has it been deprecated in favor of something else/better? Is there a reason (not) to use?

Thanks for the clarification!

**Comment by MylesBorins** [comment]
Hey @mbrevda. This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. Best of luck!

**Reply by mbrevda** [reply]
Thanks, my question is regarding the site/documentation. In past [versions](https://github.com/npm/documentation/blob/e82edcdf217786f47f10fc4e1fc667f59219cdb5/content/cli/v6/configuring-npm/package-locks.md), using the driver was recommended - it's unclear if this is no longer recommended or if an alternative strategy exists.

**Reply by MylesBorins** [reply]
Apologies I thought that this was about a non npm managed project, I'll ping the CLI team
